### PR TITLE
feat: add blog-guild page

### PR DESCRIPTION
**Pull request by petermakowski**
## Done
add blog guild page with a list of blogs (more content to come)

## QA
Go to /blog-guild and verify the page loads correctly

## Screenshots
image

### CODE REVIEW

File: templates/blog-guild.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  </div>
+</section>
+
+<section class="p-strip--light is-shallow is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h3 class="p-muted-heading u-sv2">Join the Blog Guild</h3>
+      <p class="p-heading--3">Join us on our Mattermost channel</p>
+      <p>
+        <a href="" class="p-button u-no-margin--bottom">
```
Comment (anthonydillon) [inline templates/blog-guild.html]: Missing link

File: templates/blog-guild.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <div class="p-divider u-no-margin--bottom">
+        <h2 class="u-sv3">Our blogs</h2>
+        <div class="row">
+          {% for blog in blogs %}
+          <div class="col-4 p-card">
+            <div class="p-media-object--small">
+              <div class="p-media-object__details">
+                <h3 class="p-media-object__title">
+                  {{ blog.Owner }}
+                </h3>
```
Comment (anthonydillon) [inline templates/blog-guild.html:24]: Should the text be blog title?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Some questions

File: templates/blog-guild.html (hunk 2 above)
Comment (petermakowski) [reply templates/blog-guild.html:24]: Hmm... potentially. Decided to use a name for the first draft of this page. Most people just use their name as the blog title anyway.

**Comment by nottrobin** [comment]
Where do I get a `PRIVATE_KEY_ID`?

**Comment by nottrobin** [comment]
Let's Get This Merged (LGTM)

**Comment by nottrobin** [comment]
Demo here: https://webteam-canonical-com-118.demos.haus/blog-guild

**Pull request merged**